### ISSUE TITLE
2-class pytorch model in foolbox assertion Error

### ISSUE THREAD

**Issue by rose050244** [bethgelab/foolbox#141]
I use a trained pytorch model-resnet18 in foolbox. The model is trained to classify only 2 classes, and first loaded in python. But when I use it to make adversarial examples, it reported assertion Error. I think there are two possibilities: 1. data transformation 2. foolbox cannot generate adversarial examples for a model with only 2 classes. But I'm not sure. Is there anybody who know the answer?
the code is as follows:

    pmodel.eval()
    fmodel = foolbox.models.PyTorchModel(pmodel, bounds = (0, 255), num_classes = 2)
    attack = foolbox.attacks.FGSM(fmodel)

    transform = transforms.Compose([
           transforms.RandomResizedCrop(224),
           transforms.RandomHorizontalFlip(),
           transforms.ToTensor()
       ])
    img=Image.open("/home/xxx/n04254777/ILSVRC2012_val_00049561.JPEG")
    transformed_img = transform(img)
    input_img = transformed_img.numpy()
    label = 1
    adversarial = attack(input_img, label)

the reported error:
  File "/home/qingli.guo/anaconda3/lib/python3.6/site-packages/foolbox/models/pytorch.py", line 70, in batch_predictions
    assert predictions.shape == (n, self.num_classes())
AssertionError

**Comment by jonasrauber** [comment]
In the two class case, the model is currently  still expected to output two values.

**Comment by jonasrauber** [comment]
Closing because there has been no activity. Please reopen if the problem still exists.
